peachjam/views/courts.py

```diff
@@ -150,78 +90,150 @@ def grouped_judgments(self, documents):
         return [{"key": key, "judgments": list(group)} for key, group in groups]
 
 
-class CourtYearView(CourtDetailView):
+class CourtDetailView(BaseJudgmentFilterView):
+    template_name = "peachjam/court_detail.html"
+
+    @property
```

Comment:
Use `@cached_property` so we go to the db only once

Reply:
added